server/routes/friends/friend.js

```diff
@@ -0,0 +1,326 @@
+const express = require("express");
+const router = express.Router();
+const User = require("../../models/User");
+
+const authentication = require("../../middleware/authentication");
+
+//@route            POST /api/friend/:id
+//@desc             Make this user a friend of the current user
+//@access           Private
+router.post("/friend/:id", [authentication], async function (req, res) {
```

Comment:
```suggestion
router.post("/friends, [authentication], async function (req, res) { 
```
With the friend_id coming from the body